Source/Scene/Camera.js

```diff
@@ -2738,6 +2716,7 @@ define([
     Camera.clone = function(camera, result) {
         if (!defined(result)) {
             result = new Camera(camera._scene);
+            result.frustum = camera.frustum.clone(); // Allocate a new frustum of the correct type
```

Comment:
I added this back for completeness. It's a bit wasteful since Camera already creates a frustum. Maybe the frustum could be Camera's second param, or maybe it doesn't matter.